datadog/tests/resource_datadog_powerpack_alert_graph_test.go

```diff
@@ -0,0 +1,50 @@
+package test
+
+import (
+	"testing"
+)
+
+const datadogPowerpackAlertGraphTest = `
+resource "datadog_powerpack" "alert_graph_powerpack" {
+	name         = "{{uniq}}"
+    tags = ["tag:foo1"]
+	description   = "Created using the Datadog provider in Terraform"
+	template_variables {
+		defaults = ["defaults"]
+		name     = "datacenter"
+	}
+    widget {
+      alert_graph_definition {
+        alert_id  = "895605"
```

Comment:
**question:** Can we test for something like `datadog_monitor.foo.id`, rather than this hardcoded ID?

For example, something like this:
https://github.com/DataDog/terraform-provider-datadog/blob/8ec2dd31bdfee088ba692abd42ce7b69197b7243/datadog/tests/resource_datadog_downtime_test.go#L576-L601

Reply:
Thanks! 🙌